docs/antrea-network-policy.md

```diff
@@ -689,6 +690,61 @@ The following kubectl commands can be used to retrieve CG resources:
     kubectl get cg.core.antrea.tanzu.vmware.com
 ```
 
+## Select Namespace by Name
+
+Kubernetes NetworkPolicies and Antrea-native NetworkPolicies allow selecting
+workloads from Namespaces with the use of a label selector (i.e. `namespaceSelector`).
+However, it is often desired that the Namespaces be selected directly by its `name`
+as opposed to using the `labels` associated with the Namespace. In order to select
+Namespaces by name, Antrea labels Namespaces with a reserved label `antrea.io/metadata.name`,
+which sets it's value to the name of the Namespace. Users can then use this label in the
+`namespaceSelector` field, in both, K8s NetworkPolicies and Antrea-native policies to
+select Namespaces by name. By default, Namespaces are not labeled with reserved name label.
+In order for Antrea controller to label the Namespaces, the `labelsMutator`
+`MutatingWebhookConfiguration` must be enabled. This can be done by generating and applying
+the `antrea.yaml` deployment manifest in the following way:
+
+```bash
+    # From "hack" directory
```

Comment:
no need for leading spaces?

maybe just replace both lines with `./hack/generate-manifest.sh --custom-adm-controller`